### DISCUSSION TITLE
--param force-lavd-decoder no longer valid

### DISCUSSION THREAD

**Discussion by alatteri** [CESNET/UltraGrid#236] [General]
force-lavd-decoder no longer seems to be a valid parameter.  I am unsure how I am supposed to force decoding offload to an Nvidia card (hevc_cuvid).  

I don't see an alternative to enabling this.

```
[2022-07-16 19:57:56] Unknown parameter: force-lavd-decoder
[2022-07-16 19:57:56] Type 'uv --param help' for list.
[2022-07-16 19:57:56] common_preinit() failed!
```

**Comment by alatteri** [comment]
and it is still in some parts of the code though...

from `libavcodec.c`

```
ADD_TO_PARAM("force-lavd-decoder", "* force-lavd-decoder=<decoder>[:<decoder2>...]\n"
                "  Forces specified Libavcodec decoder. If more need to be specified, use colon as a delimiter.\n"
                "  Use '-c libavcodec:help' to see available decoders.\n");
```

**Comment by MartinPulec** [comment]
Thanks, I've converted this to issue https://github.com/CESNET/UltraGrid/issues/237, please see there.

**Reply by alatteri** [reply]
Thank you.
